src/app/modules/time-entries/pages/time-entries.component.spec.ts

```diff
@@ -18,6 +18,7 @@ import { TimeEntriesComponent } from './time-entries.component';
 import { ActionsSubject } from '@ngrx/store';
 import { EntryActionTypes } from './../../time-clock/store/entry.actions';
 import { NgxMaterialTimepickerModule } from 'ngx-material-timepicker';
+import { months } from 'moment';
```

Comment:
In which part of this file do you use this "months" of moment

Reply:
I'm not using it, I'm going to delete this line